hieradata_aws/class/production/rabbitmq.yaml

```diff
@@ -0,0 +1,3 @@
+---
+
+govuk::node::s_rabbitmq::apps_using_rabbitmq: []
```

Comment:
minor: newline missing (we seem to have a general standard of caring about this in govuk)